### DISCUSSION TITLE
Unable to install R Package

### DISCUSSION THREAD

**Discussion by ongjunyi96** [BrainEnergyLab/Inflammation-Index#25] [R Package Issues]
Hi! I've managed to run through the imagej portion, but have face 2 issues: Firstly I do not get the scholl plots after mask quantification (but I get all the masks)
secondly when trying to install the R package, i get this error:
Warning in file.copy(pkgname, Tdir, recursive = TRUE, copy.date = TRUE) :
     problem copying .\R Package\InflammationIndex.Rcheck\00_pkg_src\InflammationIndex\Microglia Morphology Analysis Plugin - ImageJ\Microglia_Morphology_Analysis_Plugin_Sholl_Analysis_Script.py to C:\Users\ong_j\AppData\Local\Temp\RtmpYlqy7M\Rbuild684c284728a2\R Package\InflammationIndex.Rcheck\00_pkg_src\InflammationIndex\Microglia Morphology Analysis Plugin - ImageJ\Microglia_Morphology_Analysis_Plugin_Sholl_Analysis_Script.py: No such file or directory
    ERROR
   copying to build directory failed
Error: Failed to install 'InflammationIndex' from GitHub:
  ! System command 'Rcmd.exe' failed

am i lacking some prerequisites? from what i understand this is me not having the .py script, but i've checked and it's in the temp folder, and also in the imagej script folder. I've also tried putting the .py in the parent (/plugins) instead of child (/script) and with the same result!

**Comment by mkmarshall0** [comment]
Hello, I'm having the same issue with the sholl results - did you ever find a solution to this?

**Comment by charliefriend** [comment]
Hi, I'm also having the same issue when trying to install the inflammation index package. Is there any fix for this?

**Comment by Libharriss** [comment]
Hi, did you manage to fix this issue and run the R code all the way through? I cloned the R package into my own files and then installing the package on R worked. I have the sholl files but then the R code (morphpreprocessing step) cannot find the cell parameters and sholl csv files
